### PR TITLE
fix single network wiki

### PR DESCRIPTION
**Pull request by williamaronli**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by frankbu** [comment]
Unclear how this doc relates to the other VM install doc in the setup section. Requesting @istio/wg-environments-maintainers sort out the VM documents.

**Comment by frankbu** [comment]
IIUC, this is broken in the current 1.7 docs. We should try to get this fixed ASAP.

Can somebody from @istio/wg-environments-maintainers review this PR and help to get it merged?

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 1]
```diff
@@ -40,34 +40,18 @@ The following instructions:
 - Use Google platform-specific commands for some steps.
 
 ## Installation steps
-
+Determine and store the IP address of the Istiod since the VMs
```
Comment (frankbu) [inline content/en/docs/examples/virtual-machines/single-network/index.md:43]:
```suggestion

```

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 2]
```diff
@@ -140,60 +158,71 @@ following commands on a machine with cluster admin privileges:
     As an example, for very simple demo setups, you can also use:
 
     {{< text bash >}}
-    $ go run istio.io/istio/security/tools/generate_cert \
-          -client -host spiffee://cluster.local/vm/vmname --out-priv key.pem --out-cert cert-chain.pem  -mode citadel
-    $ kubectl -n istio-system get cm istio-ca-root-cert -o jsonpath='{.data.root-cert\.pem}' > root-cert.pem
+        $ kubectl -n "${SERVICE_NAMESPACE}" get configmaps istio-ca-root-cert -o json | jq -j '."data"."root-cert.pem"' > root-cert
```
Comment (frankbu) [inline content/en/docs/examples/virtual-machines/single-network/index.md:161]:
```suggestion
    $ kubectl -n "${SERVICE_NAMESPACE}" get configmaps istio-ca-root-cert -o json | jq -j '."data"."root-cert.pem"' > root-cert
```

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 3]
```diff
@@ -140,60 +158,71 @@ following commands on a machine with cluster admin privileges:
...
     {{< /text >}}
 
 ### Setting up the VM
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Copy the previously created `cluster.env` and `*.pem` files to the VM. For example:
+1.  Copy the previously created `cluster.env`,`root-cert`,`istio-token`, and `sidecar.env` to the VM. For example:
```
Comment (frankbu) [inline content/en/docs/examples/virtual-machines/single-network/index.md:168]:
```suggestion
1.  Copy the previously created `cluster.env`, `root-cert`, `istio-token`, and `sidecar.env` to the VM. For example:
```

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 4]
```diff
@@ -140,60 +158,71 @@ following commands on a machine with cluster admin privileges:
...
     {{< /text >}}
+    
+1. Prepare some paths for Envoy
+
+     {{< text bash >}}
+     $ sudo mkdir -p /etc/istio/proxy
+     {{< /text >}}
 
-1.  Transfer ownership of the files in `/etc/certs/` , `/var/lib/istio/envoy/` and `/var/run/secrets/istio/`to the Istio proxy.
+1. Transfer ownership of the files in `/var/lib/istio`,`/etc/certs/`,`/etc/istio/proxy` and `/var/run/secrets` to the Istio proxy:
```
Comment (frankbu) [inline content/en/docs/examples/virtual-machines/single-network/index.md:221]:
```suggestion
1. Transfer ownership of the files in `/var/lib/istio`, `/etc/certs/`, `/etc/istio/proxy` and `/var/run/secrets` to the Istio proxy:
```

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 5]
```diff
@@ -250,15 +279,49 @@ The `server: envoy` header indicates that the sidecar intercepted the traffic.
     $ echo ${GCE_IP}
     {{< /text >}}
 
-1. Add VM services to the mesh
+1. Create a service account
```
Comment (frankbu) [inline content/en/docs/examples/virtual-machines/single-network/index.md:282]:
```suggestion
1. Create a service account
```

**Comment by frankbu** [comment]
@williamaronli Please resolve the merge conflicts and fix the formatting and lint errors. Thanks.

**Comment by frankbu** [comment]
Can somebody from @istio/wg-environments-maintainers please finish fixing this doc if @williamaronli can't

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 6]
```diff
@@ -128,6 +125,27 @@ following commands on a machine with cluster admin privileges:
     {{< text bash >}}
     $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> cluster.env
     {{< /text >}}
+    
+1. Generate a `sidecar.env` to enable the cert rotation. The file contains
+   the directory path to store the cert and update the cert for cert rotation
+
+    {{< text bash >}}
+    $ echo "PROV_CERT=/var/run/secrets/istio" >> sidecar.env
```
Comment (sdake) [inline content/en/docs/examples/virtual-machines/single-network/index.md:133]: does something rebuild these files on a reboot? /var/run is a tmpfs and reset every reboot.

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 7]
```diff
@@ -140,60 +158,71 @@ following commands on a machine with cluster admin privileges:
...
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Copy the previously created `cluster.env` and `*.pem` files to the VM. For example:
+1.  Copy the previously created `cluster.env`,`root-cert`,`istio-token`, and `sidecar.env` to the VM. For example:
 
     {{< text bash >}}
     $ export GCE_NAME="your-gce-instance"
-    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
+    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {cluster.env,root-cert,sidecar.env,istio-token} ${GCE_NAME}:~
```
Comment (sdake) [inline content/en/docs/examples/virtual-machines/single-network/index.md:172]: no platform specific instructions in the documentaiton please. instead - recommend ssh to copy the files.

File: content/en/docs/examples/virtual-machines/single-network/index.md [hunk 8]
```diff
@@ -140,60 +158,71 @@ following commands on a machine with cluster admin privileges:
...
     {{< text bash >}}
     $ gcloud compute ssh --project=${MY_PROJECT} --zone=${MY_ZONE} "${GCE_NAME}"
-    $ curl -L https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
     $ sudo dpkg -i istio-sidecar.deb
     {{< /text >}}
 
-1.  Add the IP address of the Istiod to `/etc/hosts`. Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the IP address.
-The following example updates the `/etc/hosts` file with the Istiod address:
+1.  Add the Ingress Gateway IP address of the Istiod to `/etc/hosts`. Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the Ingress Gateway IP address.
```
Comment (sdake) [inline content/en/docs/examples/virtual-machines/single-network/index.md:183]: does Ingress Gatway need to be capitalized cc @frankbu

**Review by sdake — COMMENTED** [review]
This seems like the wrong content. We already have an installing virtual machines document. What we need in the `examples` path are example applications that use the installing virtual machines docume not, not replacing it.

I'd recommend two different example  documents:
1. something simple like sleep or echo
2. something more complex like bookinfo.

Id do 1 first, then 2.

In the meantime, I believe we should remove both of these example documents, but keep the virtual machine installation instructions under setup.

**Comment by sdake** [comment]
@williamaronli I have reworked this PR at @frankbu's request here: https://github.com/istio/istio.io/pull/8203.

**Comment by frankbu** [comment]
Superseded by #8203

**Pull request closed without merging**